conans/test/functional/generators/components/test_components_cmake_generators.py

```diff
@@ -500,15 +499,19 @@ def package_info(self):
         assert ("Component 'greetings::non-existent' not found in 'greetings' "
                 "package requirement" in client.out)
 
-    @pytest.mark.parametrize("generator", ["cmake_find_package_multi", "cmake_find_package"])
-    def test_component_not_found_cmake(self, generator):
+    @pytest.mark.parametrize("generator, filename", [("cmake_find_package_multi", "greetings"),
```

Comment:
I have extended this test to cover cases where filename is not the same as the name of the package, because it was failing for those cases: https://github.com/conan-io/conan/pull/10077/files#r757272609